testing/it/e2e-selenium/src/test/java/com/adobe/cq/wcm/core/components/it/seljup/components/teaser/v1/Teaser.java

```diff
@@ -32,6 +33,7 @@
     private static String teaserDescription = ".cmp-teaser__description";
     private static String teaserActionLink = "a.cmp-teaser__action-link";
     private static String imageTag = teaserImage + " img[src*='%s/_jcr_content/root/responsivegrid/teaser']";
+    public static String teaserWithAtlText = ".cmp-image__image[src*='%s/_jcr_content/root/responsivegrid/teaser.coreimg.'][alt='%s']";
```

Comment:
`teaserWithAltText`

Reply:
@richardhand Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1828/commits/484eef1b066e43e55d44e6ffd343f556de31bccb